feedback/feedback.py

```diff
@@ -415,3 +420,79 @@ def is_staff(self):
         else:
             # In workbench and similar settings, always return true
             return True
+
+
+@XBlockAside.needs('completion')
+class FeedbackXBlockAside(FeedbackXBlock, XBlockAside):
+    """
+    XBlock Aside version of FeedbackXBlock
+    """
+
+    # The scope user_state_summary is not supported in XBlockAside unless
+    # this XBlockAside is limited to a subsection. This is because the
+    # user_state_summary means the key used is the unit ID.
+    #vote_aggregate = List(
+    #    default=None, scope=Scope.user_state,
+    #    help=_("A list of user votes")
+    #)
```

Comment:
The scope `user_state_summary` is not compatible with the XBlockAside as the fields are not cached.

This is because the fields are behind a write through cache that is not prefetch correctly due to incorrect implementations in edx-platform. More context here: https://github.com/openedx/edx-platform/pull/33554